iotile_ext_cloud/iotile/cloud/cloud.py

```diff
@@ -88,6 +89,14 @@ def device_info(self, device_id):
 
         return dev
 
+    @return_type("bool")
+    def check_time(self):
+        """Checks if the current system time is consistent with iotile.cloud time
```

Comment:
minor style: docstring format should have a period after the short summary and the closing `"""` on the same line if the docstring is a single line.  Also, should be in imperative style (Check if current system time...) rather than descriptive (checks if current system time...)